ui/mirage/factories/dynamic-host-volume.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { Factory } from 'ember-cli-mirage';
+import faker from 'nomad-ui/mirage/faker';
+import { pickOne } from '../utils';
+import { dasherize } from '@ember/string';
+
+export default Factory.extend({
+  id: (i) => `${dasherize(faker.hacker.noun())}-${i}`.toLowerCase(),
+  name() {
+    return this.id;
+  },
```

Comment:
The DHV field `ID` is always a UUID. So we should probably use a UUID because they're really long and that'll influence how the page looks.

The human-friendly non-unique name is the `Name` field, which we're missing here.